### ISSUE TITLE
Passkeys support

### ISSUE THREAD

**Issue by radekwlsk** [ferdium/ferdium-app#1487]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement+%3Asparkles%3A%22) for a feature request that matches the one I want to file, without success.

### Problem Description

My organization switched 2FA from one-time codes to Passkeys only and I can't log in to any company service in Ferdium ever since.

It would be nice to be able to continue using it but without passkeys support it is becoming less and less useful.

More on passkeys: https://blog.google/technology/safety-security/the-beginning-of-the-end-of-the-password/

### Proposed Solution

Support passkeys.

### Alternatives Considered

I have no alternative.

**Comment by mtt59** [comment]
I think this has been implemented as of writing this comment

**Comment by SpecialAro** [comment]
As far as I can tell, such feature was never implemented in Ferdium.

**Comment by vypxl** [comment]
On Linux/wayland the website waits for a passkey, but no prompt is shown to actually use it. Devtools don't show an error.

**Comment by tmsjordan** [comment]
> On Linux/wayland the website waits for a passkey, but no prompt is shown to actually use it. Devtools don't show an error.

Same issue here. Linux/Wayland(Hyprland) and no passkey prompts across multiple services (Github,DockerHub, etc)

**Comment by locomoco28** [comment]
Same thing on Debian with X11/KDE

**Comment by Lockszmith-GH** [comment]
This is probably an electron based issue, not specific to ferdium.

**Comment by conny3496** [comment]
Yes, as far as I can see it does not work as of today. The promt to unlock the device will not show up.

**Comment by DennaGherlyn** [comment]
I just discovered Ferdium, was considering to use it to split my homelab admin tasks from my normal usage, but because of this issue it's sadly not useable for me, as we use Passkeys for everything.

Fedora 42/KDE/Wayland/AppImage

This is all the console shows. I triggered the pop-up after the last line of `GetVSyncParametersIfAvailable() failed` but nothing was logged.

```
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
[990490:1025/195724.151154:ERROR:ui/gl/gl_surface_presentation_helper.cc:260] GetVSyncParametersIfAvailable() failed for 1 times!
[990453:1025/195724.250855:ERROR:base/nix/mime_util_xdg.cc:137] Invalid mime.cache file does not contain null prior to ALIAS_LIST_OFFSET=44
[MobX] Since strict-mode is enabled, changing (observed) observable values without using an action is not allowed. Tried to modify: Settings@13.store
Ferdium Migration:  🧙‍  Running database migration wizard
info: serving app on http://localhost:46569
(node:990453) [DEP0180] DeprecationWarning: fs.Stats constructor is deprecated.
(Use `ferdium --trace-deprecation ...` to show where the warning was created)
Ferdium Migration:  🔧  Nothing to migrate, already on the newest version
[990490:1025/195729.202975:ERROR:ui/gl/gl_surface_presentation_helper.cc:260] GetVSyncParametersIfAvailable() failed for 2 times!
[990490:1025/195729.209385:ERROR:ui/gl/gl_surface_presentation_helper.cc:260] GetVSyncParametersIfAvailable() failed for 3 times!
Checking for update
Update for version 7.1.1 is not available (latest version: 7.1.1, downgrade is disallowed).
```

**Comment by spthiel** [comment]
With 7.1.2 this behaviour is still present

@SpecialAro why was this closed?
This seems to be waiting for #2337 or is there a setting to enable it?
